osbs/api.py

```diff
@@ -827,10 +827,13 @@ def get_orchestrator_build_logs(self, build_id, follow=False, wait_if_missing=Fa
         :param follow: bool, fetch logs as they come?
         :param wait_if_missing: bool, if build doesn't exist, wait
         :return: generator yielding objects with attributes 'platform' and 'line'
+                 None, if logs doesn't exist
```

Comment:
Technically it returns a generator which yields no objects. In other words, the caller can always iterate over the value returned, but this line makes it appear as though they would need to check for identity with `None` before iterating.

Reply:
In that case I should change the return statement to return an empty generator?

Reply:
No. The method is already a generator function because it uses the `yield` keyword. This is different from a regular function in that calling the function immediately returns a generator. When the caller iterates over the generator, the called function runs up until the next `yield` statement.

When a generator function returns None, this means the iteration stops. The caller will see an empty iterator.

So what I described in the comment above is already the behaviour your change has, and in fact is exactly the behaviour we'd like to see... just the docstring doesn't describe it correctly.